src/style.css

```diff
@@ -0,0 +1,76 @@
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  display: flex;
+  width: 96px;
+  height: 96px;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  align-self: auto;
+
+}
+
+.stars__star {
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(n),
+.stars--2 .stars__star:nth-child(n),
+.stars--3 .stars__star:nth-child(n),
+.stars--4 .stars__star:nth-child(n),
+.stars--5 .stars__star:nth-child(n) {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 .stars__star:nth-child(n) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+
+}
```

Comment:
Accodring to chechlist you can combine this selectors using comma
![image](https://github.com/mate-academy/layout_stars/assets/80700247/6fdc2cb8-e746-418a-be8c-41d47c81b262)
```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
  background-repeat: no-repeat;
  background-position: center;
}
```

Reply:
* Not Fixed